src/dmap_common.c

```diff
@@ -355,7 +355,7 @@ dmap_send_error(struct evhttp_request *req, const char *container, const char *e
     {
       DPRINTF(E_LOG, L_DMAP, "Could not allocate evbuffer for DMAP error\n");
 
-      evhttp_send_error(req, HTTP_SERVUNAVAIL, "Internal Server Error");
+      httpd_send_error(req, HTTP_SERVUNAVAIL, "Internal Server Error");
```

Comment:
dmap_send_error uses evhttp_send_reply, which won't include the headers. So with the changes you have made here, dmap_send_error will only include the headers on these rare out of memory edge cases. I don't understand the point of that.